src/aap_eda/services/activation/manager.py

```diff
@@ -507,7 +494,7 @@ def _failed_policy(self):
         ):
             LOGGER.info(
                 f"Activation id: {self.db_instance.id} "
-                f"Restart policy is set to {self.db_instance.restart_policy}. ",
+                f"Restart policy is set to {self.db_instance.restart_policy}. "
                 "Has reached the maximum number of restarts. ",
             )
```

Comment:
Adding a definitive statement the activation failed to start is preferable to leaving the user to have to infer it (regardless of how "obvious" such an inference would be).

Reply:
Agree